### PR TITLE
Updated version to 8.16

### PR DESCRIPTION
**Pull request by apollo13**
I verified locally that audio calling and chatting works.

### CODE REVIEW

**Comment by jurf** [comment]
bot, build

File: com.skype.Client.appdata.xml [hunk 1]
```diff
@@ -25,6 +25,6 @@
   <icon type="remote" height="64" width="64">https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w64</icon>
   <icon type="remote" height="128" width="128">https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w128</icon>
   <releases>
-    <release date="2018-02-01" version="8.15.76.2"/>
+    <release date="2018-02-09" version="8.16.76.1"/>
```
Comment (jurf) [inline com.skype.Client.appdata.xml]: @TingPing so no stacking?

Comment (Barthalion) [reply com.skype.Client.appdata.xml]: For now yes, as the only use is displaying version. (Although I still see some value but that's not the place for discussion about it.)

**Comment by Barthalion** [comment]
Thanks!

**Pull request merged**